### PR TITLE
Enhancement of career page

### PR DESCRIPTION
**Pull request by yugandhar-pathi**
@KalyaniPayyavula .. update screenshots here.

### CODE REVIEW

File: Career/career.html [hunk 1]
```diff
@@ -2,24 +2,68 @@
...
+            <button id="deleteFieldTwo">Delete</button>
+        </div>
+        <dialog id="dialogTwo">
+            <h1>
+                <img src="../Images/20220321113532064.jpg" alt="Skandhana" height="170px"></img><br>
+                <b>Name : Skandhana</b><br>
+                <b>Grade : 1st Grade</b>
+                <button onclick="closeDialogTwo()">Close</button>
+            </h1>
+        </dialog>
```
Comment (yugandhar-pathi) [inline Career/career.html]: You need to be smart !! Don't hardcode dialog code like this. You can make it dynamic.

File: Career/career.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    dialogTwo.close()
+}
+
+const clickLiThree = () => {
+    const dialogThree = document.getElementById('dialogThree')
+    dialogThree.showModal()
+}
+const closeDialogThree = () => {
+    dialogThree.close()
+}
```
Comment (yugandhar-pathi) [inline Career/career.js]: Remove all this. Make listener dynamic.

File: Career/career.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+const deleteFieldOne = () => {
+    document.getElementById('liIdOne').parentNode.removeChild(liIdOne)
+}
+const deleteFieldTwo = () => {
+    document.getElementById('liIdTwo').parentNode.removeChild(liIdTwo)
+}
+const deleteFieldThree = () => {
+    document.getElementById('liIdThree').parentNode.removeChild(liIdThree)
+}
```
Comment (yugandhar-pathi) [inline Career/career.js]: You need to make this dynamic as well.

File: Career/career.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+document.getElementById('addButton').addEventListener('click', addStudentFields)
+document.getElementById('deleteFieldOne').addEventListener('click', deleteFieldOne)
+document.getElementById('deleteFieldTwo').addEventListener('click', deleteFieldTwo)
+document.getElementById('deleteFieldThree').addEventListener('click', deleteFieldThree)
```
Comment (yugandhar-pathi) [inline Career/career.js]: You need to make this dynamic.

File: Career/career.html [hunk 5]
```diff
@@ -2,24 +2,68 @@
...
+            <button id="deleteFieldThree">Delete</button>
+        </div>
+        <dialog id="dialogThree">
+            <h1>
+                <img src="../Images/20220321113910381.jpg" alt="Kalyani" height="170px"></img><br>
+                <b>Name : Kalyani</b><br>
+                <b>Grade : Kinder Garden</b>
+                <button onclick="closeDialogThree()">Close</button>
+            </h1>
+        </dialog>
```
Comment (yugandhar-pathi) [inline Career/career.html]: Same here.

File: Career/career.html [hunk 6]
```diff
@@ -2,24 +2,68 @@
...
+            <button id="deleteFieldOne">Delete</button>
+        </div>
+
+        <dialog id="dialogOne">
+            <h1> <img src="../Images/20220321113235643.jpg" alt="Reshitha" height="170px"></img><br>
+                <b>Name : Reshitha</b><br>
+                <b>Grade : 2nd Grade</b>
+                <button onclick="closeDialogOne()">close</button>
+            </h1>
+        </dialog>
```
Comment (yugandhar-pathi) [inline Career/career.html]: Remove hardcoding of dialog code.

**Review by yugandhar-pathi — COMMENTED** [review]
Fix all review comments.

**Review by KalyaniPayyavula — APPROVED** [review]

**Pull request merged**